commit 3b8f14b41026fb7d7e9a4af2a4128a702d07ad26
Author: Adrian Bunk <bunk@kernel.org>
Date:   Sat Jul 26 15:22:28 2008 -0700

    mm/util.c must #include <linux/sched.h>
    
    mm/util.c: In function 'arch_pick_mmap_layout':
      mm/util.c:144: error: dereferencing pointer to incomplete type
      mm/util.c:145: error: 'arch_get_unmapped_area' undeclared (first use in this function)
      mm/util.c:145: error: (Each undeclared identifier is reported only once
      mm/util.c:145: error: for each function it appears in.)
      mm/util.c:146: error: 'arch_unmap_area' undeclared (first use in this function)
    
    Signed-off-by: Adrian Bunk <bunk@kernel.org>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/mm/util.c linux-source-2.6.26/mm/util.c
--- linux-source-2.6.26.orig/mm/util.c	2010-01-29 17:50:35.000000000 -0700
+++ linux-source-2.6.26/mm/util.c	2010-01-31 15:41:53.000000000 -0700
@@ -6,6 +6,7 @@
 #include <linux/syscalls.h>
 #include <linux/mman.h>
 #include <linux/file.h>
+#include <linux/sched.h>
 #include <asm/uaccess.h>
 
 /**
